docs/data-sources/host.md

```diff
@@ -0,0 +1,23 @@
+# xenorchestra_host
+
+Provides information about a host.
+
+## Example Usage
+
+```hcl
+data "xenorchestra_host" "host1" {
+  name_label = "Your host"
+}
+resource "xenorchestra_vm" "node" {
+    //...
+    affinity_host = data.xenorchestra_host.ng6.id
+    //...
+}
+
+```
+
+## Argument Reference
+* name_label - (Required) The name of the host you want to look up.
```

Comment:
Regarding my comment about checking the number of hosts returned. We will need to make sure the documentation calls out that this will error if there are more than one host returned (see the "note" on https://github.com/terra-farm/terraform-provider-xenorchestra/blob/master/docs/data-sources/cloud_config.md)

Reply:
Added **NOTE**